src/navigation/NavigationContainer.tsx

```diff
@@ -0,0 +1,109 @@
+import {
+  NavigationContainer as NativeNavContainer,
+  NavigatorScreenParams,
+} from "@react-navigation/native";
+import {
+  createDrawerNavigator,
+  DrawerScreenProps,
+} from "@react-navigation/drawer";
+import { HeaderButtonProps } from "@react-navigation/native-stack/lib/typescript/src/types";
+import {
+  createNativeStackNavigator,
+  NativeStackNavigationOptions,
+} from "@react-navigation/native-stack";
+
+import { Home } from "../screens/Home";
+import { Bar } from "../screens/sync/Bar";
+import { ProjectCoordinator } from "../screens/projectInvite/ProjectCoordinator";
+import { WHITE } from "../lib/styles";
+import { CustomHeaderLeft } from "../components/CustomHeaderLeft";
+import { CustomHeaderRight } from "../components/CustomHeaderRight";
+
+export type Drawers = {
+  Home: undefined;
+  Sync: NavigatorScreenParams<SyncScreens>;
+  ProjectInvite: NavigatorScreenParams<ProjectInviteScreens>;
+};
+
+export type SyncScreens = {
+  Bar: undefined;
+};
+
+export type ProjectInviteScreens = {
+  ProjectCoordinator: undefined;
+};
```

Comment:
See [Type Checking With Typescript](https://reactnavigation.org/docs/typescript) in `React Nav` Docs